### ISSUE TITLE
Get lastUpdated of browscap

### ISSUE THREAD

**Issue by ThaDafinser** [browscap/browscap-php#132]
It's currently possible to retrieve the used `browscap version` from cache (e.g. 6010)

Is it somehow possible to also get the release date of that browscap file?

```php
$version = $parser->getCache()->getVersion(); // 6010
$lastUpdate = $parser->getCache()->getLastUpdate(); // DateTime or date string of the creation of the browscap file
```

The date to be used would be this from browscap.ini
```
[GJK_Browscap_Version]
Version=6010
Released=Sun, 22 Nov 2015 15:41:15 +0000
```

If you have no internet connection, you cannot use this method and you cant check with that method, what's your local timestamp (or that timestamp is simply wrong, since it's the date when you added the data file)
```php
$parser->getLoader()->getMTime();
```

**Comment by ThaDafinser** [comment]
For example i grab it from wurfl like this: https://github.com/ThaDafinser/UserAgentParser/blob/master/src/Provider/Wurfl.php#L99-L112

**Comment by mimmi20** [comment]
@ThaDafinser Actually we dont need this date in the project. If you like to have, you are free to implement the functionality and create a PR for that.

**Comment by asgrim** [comment]
Fixed by PR #133 and #138
